Modules/Filtering/FastMarching/include/itkFastMarchingUpwindGradientImageFilter.hxx

```diff
@@ -67,15 +67,20 @@ FastMarchingUpwindGradientImageFilter<TLevelSet, TSpeedImage>::VerifyPreconditio
 
   switch (this->m_TargetReachedMode)
   {
-    case AllTargets:
+    case TargetConditionEnum::AllTargets:
       this->VerifyTargetReachedModeConditions(this->m_NumberOfTargets);
       break;
-    case OneTarget:
+    case TargetConditionEnum::OneTarget:
       this->VerifyTargetReachedModeConditions(1);
       break;
-    case SomeTargets:
+    case TargetConditionEnum::SomeTargets:
       this->VerifyTargetReachedModeConditions(this->m_NumberOfTargets);
       break;
+    case TargetConditionEnum::NoTargets:
```

Comment:
@blowekamp Had to add this case to make gcc happy:
https://open.cdash.org/viewBuildError.php?type=1&buildid=8396896

I also added a `default` case. A review would be appreciated.